AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/TestSupport/BaseDataStoreTests.swift

```diff
@@ -70,6 +71,7 @@ class BaseDataStoreTests: XCTestCase {
             try Amplify.add(plugin: apiPlugin)
             try Amplify.add(plugin: dataStorePlugin)
             try Amplify.configure(amplifyConfig)
+            XCTAssertEqual(dataStorePlugin.dispatchedModelSyncedEvents.count, 7)
```

Comment:
`TestModelRegistration()` contains 7 models that are registered on configure. to make it more clear, i can assert against ModelRegistry.modelSchemas.count rather than a hardcoded value here